pkg/sql/catalog/descpb/structured.proto

```diff
@@ -1062,14 +1064,14 @@ message TableDescriptor {
                                          (gogoproto.casttype) = "ID"];
     }
 
-    optional SequenceOwner sequence_owner = 6 [(gogoproto.nullable) = false];
+    optional SequenceOwner sequence_owner = 7 [(gogoproto.nullable) = false];
 
     // The number of values (which have already been created in KV)
     // that a node can cache locally.
-    optional int64 cache_size = 7 [(gogoproto.nullable) = false];
+    optional int64 cache_size = 8 [(gogoproto.nullable) = false];
     // AS option value for CREATE SEQUENCE, which specifies the default
     // min and max values a sequence can take on.
-    optional string as_integer_type = 8 [(gogoproto.nullable) = false];
+    optional string as_integer_type = 9 [(gogoproto.nullable) = false];
```

Comment:
we should not renumber protobufs fields - otherwise it breaks compatibility with older versions (e.g. older versions expect as_integer_type = 8, but you've changed it to nine so they are broken).
use `restart = 10` instead - the order in which they appear does not matter, and revert the other changes.

Reply:
this is still an issue